src/mumble/Log.cpp

```diff
@@ -728,6 +724,18 @@ void Log::log(MsgType mt, const QString &console, const QString &terse, bool own
 		const int oldscrollvalue = tlog->getLogScroll();
 		const bool scroll        = (oldscrollvalue == tlog->getLogScrollMaximum());
 
+		// A newline is inserted after each frame, but this spaces out the
+		// log entries too much, so the line height is set to zero to reduce
+		// the space between log entries. This line height is only set for the
+		// blank lines between entries, not for entries themselves.
+		QTextBlockFormat bf = tc.blockFormat();
+		bf.setLineHeight(0, QTextBlockFormat::FixedHeight);
+		bf.setTopMargin(0);
+		bf.setBottomMargin(0);
+
+		// Set the line height of the leading blank line to zero
+		tc.setBlockFormat(bf);
```

Comment:
Am I correct in assuming that at this point the block manipulated is the "old" one - aka: the block of a previous message, which will get a new line appended to it once we add a new frame or new block below?
And as soon as we create a new frame/block the corresponding block will have a fresh set of format options with a line height > 0.

I'm trying to understand why this would affect only the desired empty line and nothing else.